Example/Backpack/StoryGroup.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2021 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+protocol StoryGroup: CaseIterable, Story {}
+
+extension StoryGroup {
+  static var allExamples: [Item] {
+    self.allCases.map({ $0.example })
+  }
+}
```

Comment:
`StoryGroup` extends story and is intended to be used with enums for when there are multiple examples in a story.